src/components/navBar/navBar.js

```diff
@@ -240,7 +245,9 @@ MdNavBarController.prototype._updateInkBarStyles = function(tab, newIndex, oldIn
   var left = tabEl.offsetLeft;
 
   this._inkbar.toggleClass('_md-left', newIndex < oldIndex)
-      .toggleClass('_md-right', newIndex > oldIndex);
+      .toggleClass('_md-right', newIndex > oldIndex)
+      .toggleClass('md-button', newIndex === -1);
```

Comment:
I do not understand this part; it is toggling a 'md-button' class on the inkbar, but there shouldn't be any such class.